config.json

```diff
@@ -69,6 +69,8 @@
     "show_accept_paypal": false,
     "show_accept_visa": false,
     "show_product_details_tabs": true,
+    "show_product_weight": true,
+    "show_product_dimensions": true,
```

Comment:
If we never displayed this info before we should default it to `false`. We don't want people calling support that new data started showing up on their product details page.

Reply:
Good point. I had thought about that but forgot to implement. Good news is I am better at squashing ;)